NuGet.internal.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<configuration>
+    <packageSources>
+        <clear />
+        <add key="DNCENG-CoreClrPal" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/CoreClrPal/nuget/v3/index.json" />
+        <add key="privateDependency" value="https://pkgs.dev.azure.com/devdiv/DevDiv/_packaging/ClrInstrumentationEngine-PrivateDependency/nuget/v3/index.json" />
```

Comment:
Will this cause issues since we need to have only 1 feed?

Reply:
From https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/secure-supply-chain/how-to-securely-configure-package-source-files, open source projects can have multiple feeds as long as they don't upstream to nuget.org, are within our control, and the packages use reserved namespaces (Microsoft.*) or are trusted authors.